### PR TITLE
Add XHProf UI

### PR DESCRIPTION
**Pull request by oxyc**
Needs work just making sure it works as I think it does.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -7,3 +7,15 @@ tideways_download_folder_name: php-profiler-extension-master
...
 # extension can be used along with the XHProf UI to view profiles.
 tideways_api_key: ''
+
+# XHProf UI
+tideways_install_xhprof_ui: yes
+
+# Use the commented URL and folder below if you want to use the 'official'
+xhprof_download_url: https://github.com/RustJason/xhprof/archive/php7.zip
+# xhprof_download_url: https://github.com/phacility/xhprof/archive/master.tar.gz
+xhprof_download_folder_name: xhprof-php7
```
Comment (oxyc) [inline defaults/main.yml]: @geerlingguy how should these be set? `tideway_xhprof_download_url` or should we maybe reuse the `xhprof_download_url` variable with a conditional set_fact and `__tideways_xhprof_download_url`?

**Comment by oxyc** [comment]
So we can actually use both the xhprof php library (tideways doesn't have any support for writing the data files) and the ui. @geerlingguy does this work for you? If it does I'll clean it up and update the README

**Comment by oxyc** [comment]
> Warning: Must specify directory location for XHProf runs. Trying /tmp as default. You can either pass the directory location as an argument to the constructor for XHProfRuns_Default() or set xhprof.output_dir ini param.

Is issued but it's only a warning and it's quite rare that someone uses the XHProf PHP lib I guess. If they do, they just have to specify the storage directory like the message says.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ The URL from which Tideways will be downloaded.
 
 If you use the Tideways UI, set this variable to your API key. Otherwise the extension can be used along with the XHProf UI to view profiles.
 
+    tideways_install_xhprof_ui: yes
+
+Tideways data-format is 100% compatible with XHProf so you can use the XHProf UI to browse profiles reports and the `XHProfRuns_Default` class to write the profile data to disk. If you use the Tideways UI, set this variable to `no`.
+
+    xhprof_download_url: https://github.com/phacility/xhprof/archive/master.tar.gz
+    xhprof_download_folder_name: xhprof-master
```
Comment (oxyc) [inline README.md:33]: As we dont compile the extension we can stick to the original `phacility/xhprof`

**Comment by geerlingguy** [comment]
I'll be testing this with Drupal VM and 7.1.

**Comment by geerlingguy** [comment]
Works great with XHProf module and OOTB Drupal VM:

screen shot 2017-01-14 at 8 32 52 pm

**Pull request merged**